embedchain/utils.py

```diff
@@ -259,6 +259,8 @@ def is_openapi_yaml(yaml_content):
     else:
         # Source is not a URL.
 
+        # TODO: check if source is gmail query
```

Comment:
Is it still a todo?

Reply:
Yes, not sure how to achieve that. Need to think more. Can add it in follow up PR.